src/components/Card/CardContent.js

```diff
@@ -17,9 +20,7 @@ const CardContent = styled.div`
   min-width: 0;
   box-sizing: border-box;
 
-  ${props => props.large && css`
-    flex: 0 0 125px;
-  `}
+  ${props => props.cardSize && CardContentLarge}
 `
 
 const Title = styled.h2`
```

Comment:
not related but not sure why `h2`, can sense? can be `h1`? I don't know

Reply:
Yeah I did a `h1` at first, but after inspecting an _existing_ embed (can't remember if Twitter or FB), they used `h2` (I guess for SEO maybe?)